kobo/rpmlib.py

```diff
@@ -69,7 +71,7 @@ def get_rpm_header(file_name, ts=None):
             ts.setKeyring(rpm.keyring())
         ts.setVSFlags(rpm._RPMVSF_NOSIGNATURES|rpm._RPMVSF_NODIGESTS)
 
-    if type(file_name) in (str, unicode):
+    if type(file_name) in (str, six.string_types):
```

Comment:
I have a regression in my application relating to this change.  Reproducer is like this:

````
python -c "import kobo.rpmlib; kobo.rpmlib.get_rpm_header(u'quux')"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/usr/lib/python2.6/site-packages/kobo/rpmlib.py", line 79, in get_rpm_header
    hdr = ts.hdrFromFdno(fo.fileno())
AttributeError: 'unicode' object has no attribute 'fileno'
````

According to docs six.string_types is basestring() in python 2.  It makes sense then, that a unicode string now fails to match this precise matching on type.

Should it instead be written like this?

````
if isinstance(file_name, six.string_types):
````

Reply:
Yes, you are right. Your solution is good for both Pythons:
- In Python 2 `six.string_types` contains `basestring()` which is a base class for `str` and `unicode`
- In Python 3 `six.string_types` contains new `str`

Fixed in the latest commit. Thank you very much.